contrib/gorm.io/gorm.v1/gorm_test.go

```diff
@@ -343,3 +343,53 @@ func TestContext(t *testing.T) {
 		assert.Equal(t, testCtx.Value(key(contextKey)), ctx.Value(key(contextKey)))
 	})
 }
+
+func TestError(t *testing.T) {
+	mt := mocktracer.Start()
+	defer mt.Stop()
+
+	assertErrCheck := func(t *testing.T, mt mocktracer.Tracer, errExist bool, opts ...Option) {
+		sqltrace.Register("pgx", &stdlib.Driver{})
+		sqlDb, err := sqltrace.Open("pgx", pgConnString)
+		if err != nil {
+			log.Fatal(err)
+		}
```

Comment:
These should probably be assert failures rather than `log.Fatal`. `log.Fatal` calls `os.Exit`, which is not desirable in test cases.

Reply:
I changed to use `assert.Nil`.